book/chapter-3/README.md

```diff
@@ -28,7 +31,9 @@ class Switcher extends React.Component {
 };
 ```
 
-That's all fine because `_handleButtonClick` is a function and we indeed pass a function to the `onClick` attribute. The problem is that as it is the code doesn't keep the same context. So, if we have to use `this` inside `_handleButtonClick` to refer the current `Switcher` component we will get an error.
+โค้ดชุดจะสามารถทำงานได้ตรงตามที่เราต้องการ เพราะ `_handleButtonClick` นั้นเป็น *function* และเราก็ส่ง *function* เข้าไปใน attribute ชื่อ `onClick`
```

Comment:
> โค้ดชุด

--> เอาคำว่า ชุด ออกจ้า

Reply:
จริงๆ เป็นคำว่า `โค้ดชุดนี้` อะครับแต่ตกคำว่า `นี้`
ให้แก้เป็น `โค้ดชุดนี้`, `โค้ดนี้` หรือ `โค้ด` เฉยๆอะครับ

Reply:
`โค้ด` or `โค้ดชุดนี้` is both fine krub